nvflare/lighter/poc.py

```diff
@@ -34,38 +32,58 @@ def clone_client(num_clients: int):
     shutil.rmtree(src_folder)
 
 
-def main():
-    parser = argparse.ArgumentParser()
-    parser.add_argument("-n", "--num_clients", type=int, default=1, help="number of client folders to create")
+def clone_poc_folder(src_poc_folder, dest_poc_folder):
+    shutil.rmtree(dest_poc_folder, ignore_errors=True)
+    try:
+        shutil.copytree(src_poc_folder, dest_poc_folder)
+    except BaseException as e:
+        print(f"Unable to copy poc folder from {src_poc_folder}, Exit. {e} ")
+        exit(1)
+    for root, dirs, files in os.walk(dest_poc_folder):
+        for file in files:
+            if file.endswith(".sh"):
+                os.chmod(os.path.join(root, file), 0o755)
 
-    args = parser.parse_args()
 
+def get_src_poc_parent():
     file_dir_path = pathlib.Path(__file__).parent.absolute()
-    poc_zip_path = file_dir_path.parent / "poc.zip"
-    poc_folder_path = file_dir_path.parent / "poc"
+    return file_dir_path.parent
```

Comment:
I personally think this function can be merged with "get_src_poc_dir".
But either way is fine.